app/test/testmerginapi.cpp

```diff
@@ -61,11 +62,59 @@ void TestMerginApi::testDownloadProject()
     ProjectList projects = mMerginProjectModel->projects();
     Q_ASSERT(!mMerginProjectModel->projects().isEmpty());
     mProjectName = mMerginProjectModel->projects().at(0)->name;
-    qDebug() << "TestMerginApi::testListProjectFinished PASSED";
+    qDebug() << "TestMerginApi::testDownloadProject PASSED";
+}
+
+void TestMerginApi::testCreateProject()
+{
+    QSignalSpy spy(mApi, SIGNAL(projectCreated()));
+    mProjectName = "TEMPORARY_TEST_PROJECT";
+    mApi->createProject(mProjectName);
+
+    QVERIFY(spy.wait(1000));
+    QCOMPARE(spy.count(), 1);
+
+    ProjectList projects = getProjectList();
+    Q_ASSERT(!mMerginProjectModel->projects().isEmpty());
+
+    bool containsTestProject = false;
+    for (std::shared_ptr<MerginProject> project: projects) {
+        if (project->name == mProjectName) containsTestProject = true;
+    }
+    Q_ASSERT(containsTestProject);
+    qDebug() << "TestMerginApi::testCreateProject PASSED";
+}
+
+void TestMerginApi::testDeleteProject()
```

Comment:
I suggest to merge the create+delete project tests together into one test function. (so there isn't just testCreateProject() method that leaves behind a test). The unified create+delete test should also check that the project does not exist already before creating it (so we are sure that create project works as expected)